implementation/leprechaun/src/route/test/route.controller.spec.ts

```diff
@@ -254,14 +304,14 @@ describe('RouteService', () => {
      */
 
     // given
-    const c1s = createTimeOffsetFromNow(1);
-    const c2s = createTimeOffsetFromNow(2);
-    const start = createTimeOffsetFromNow(3);
-    const c1e = createTimeOffsetFromNow(4);
-    const c3s = createTimeOffsetFromNow(5);
-    const end = createTimeOffsetFromNow(6);
-    const c2e = createTimeOffsetFromNow(7);
-    const c3e = createTimeOffsetFromNow(8);
+    const c1s = createTimeOffsetFromNow({ hours: 1, seconds: 0 });
+    const c2s = createTimeOffsetFromNow({ hours: 2, seconds: 0 });
+    const start = createTimeOffsetFromNow({ hours: 3, seconds: 0 });
+    const c1e = createTimeOffsetFromNow({ hours: 4, seconds: 0 });
+    const c3s = createTimeOffsetFromNow({ hours: 5, seconds: 0 });
+    const end = createTimeOffsetFromNow({ hours: 6, seconds: 0 });
+    const c2e = createTimeOffsetFromNow({ hours: 7, seconds: 0 });
+    const c3e = createTimeOffsetFromNow({ hours: 8, seconds: 0 });
```

Comment:
If seconds are optional and equal to 0, why pass them?

Reply:
I did regex replace all `createTimeOffsetFromNow` and wrote it like that.

Reply:
I'll just `createTimeOffsetFromNow\(\{ hours: (\d+), seconds: 0 \}\)` into `createTimeOffsetFromNow({ hours: $1 })`